website/content/en/docs/helm/quickstart.md

```diff
@@ -108,19 +119,22 @@ value called `replicaCount` and it is set to `1` by default. If we want to have
 2 nginx instances in our deployment, we would need to make sure our CR spec
 contained `replicaCount: 2`.
 
-Update `deploy/crds/example.com_v1alpha1_nginx_cr.yaml` to look like the following:
+Update `config/samples/example_v1alpha1_nginx.yaml` to look like the following:
 
 ```yaml
 apiVersion: example.com/v1alpha1
 kind: Nginx
 metadata:
-  name: example-nginx
+  name: nginx-sample
 spec:
+  # Default values copied from <project_dir>/helm-charts/nginx/values.yaml
+  ...
```

Comment:
The other defaults can be removed for this walkthrough, so no need to include the `...`, which may confuse things if folks are copy/pasting.
```suggestion
```